src/main/kotlin/org/taktik/icure/services/external/rest/v1/controllers/extra/TimeTableController.kt

```diff
@@ -84,6 +84,7 @@ class TimeTableController(
 
 				//Create a timeTableItem
 				val timeTableItem = TimeTableItem(
+					rrule =null,
 					calendarItemTypeId = "consult",
 					days = mutableListOf("monday"),
```

Comment:
As `days` & `recurrenceTypes` are deprecated and that you initialize rrule above, I would not give any value anymore to `days`

Reply:
I think we should'nt change anyting in the legacy system.

Reply:
Ok